### DISCUSSION TITLE
github移动端app在安卓设备预览公式部分不显示

### DISCUSSION THREAD

**Discussion by xiuxiuxius** [community/community#31059] [Mobile]
非常感谢github移动端app，平板看代码很方便。

但是，代码注释里给的数学公式无法显示，如图。

会显示出math源码，不知可否改善呢？

![Screenshot_20220828-042219_GitHub.jpg](https://user-images.githubusercontent.com/89114157/187046860-1e21b491-03c5-417f-911f-7257c838b020.jpg)

**Comment by mxie** [comment]
@xiuxiuxius 谢谢您的反馈意见！我从来没有见过这个`math`的源码。显示出来是应该什么样子？在网页浏览器上(github.com)是什么样子？

**Comment by xiuxiuxius** [comment]
感谢回复！


(谷歌)网页浏览器上(电脑和平板)显示一样。


https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/loss.py
545行







---原始邮件---
发件人: "Melissa ***@***.***&gt;
发送时间: 2022年8月30日(周二) 晚上10:22
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [community/community] github移动端app在安卓设备预览公式部分不显示 (Discussion #31059)




 
谢谢您的反馈意见！我从来没有见过这个math的源码。显示出来是应该什么样子？在网页浏览器上(github.com)是什么样子？
